helm-charts/riot-applications/templates/mongodb-client.yaml

```
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: mongodb-client
+  name: mongodb-client
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mongodb-client
+  template:
+    metadata:
+      labels:
+        app: mongodb-client
+    spec:
+      containers:
+      - image: mongodb
```

I think there is a typo, right?
```suggestion
      - image: mongo
```